### PR TITLE
Unify all docstring format

### LINKED ISSUE #68: Unify all docstring format

### PR DESCRIPTION
**Pull request by kevinshen56714**
Clean up the docstrings (remove unnecessary indentations)
closes #68

### CODE REVIEW

File: pmd/core/System.py [hunk 1]
```diff
@@ -181,10 +175,10 @@ def write_data(self, output_dir: str = '.', cleanup: bool = True) -> None:
 
         Parameters:
         output_dir (str): Directory for the generated LAMMPS data file
-                          ; default: `"."`
+            ; default: `"."`
 
         cleanup (bool): Whether to clean up files other than the LAMMPS data
-                        file PSP generated
+            file PSP generated
```
Comment (akhlakm) [inline pmd/core/System.py:181]: This sentence seems incomplete.

**Review by akhlakm — COMMENTED** [review]
Hi @kevinshen56714 I added some comments on what I think. Other than that everything looks alright.

File: pmd/core/System.py [hunk 2]
```diff
@@ -21,41 +21,35 @@ class System:
         builder (Builder): Builder (One of `EMC` or `PSP`)
 
         natoms_total (int): Total number of atoms in the system, one of
-                            this attribute and `nchains_total` has to be
-                            provided but not both (providing both will result
-                            in an error); default: `None`
+            this attribute and `nchains_total` has to be provided but not both
```
[the remark this answers is not in the document]
Comment (kevinshen56714) [reply pmd/core/System.py]: Changed to *or*

File: pmd/core/System.py (hunk 1 above)
Comment (kevinshen56714) [reply pmd/core/System.py]: Updated.

File: pmd/core/System.py [hunk 3]
```diff
@@ -344,10 +331,10 @@ def write_data(self, output_dir: str = '.', cleanup: bool = True) -> None:
 
         Parameters:
         output_dir (str): Directory for the generated LAMMPS data file
-                          ; default: `"."`
+            ; default: `"."`
 
         cleanup (bool): Whether to clean up files other than the LAMMPS data
-                        file PSP generated
+            file PSP generated
```
[the remark this answers is not in the document]
Comment (kevinshen56714) [reply pmd/core/System.py]: Updated.

**Review by akhlakm — APPROVED** [review]

**Pull request merged**